### ISSUE TITLE
ansible install doesn't set nodeselector for template service broker?

### ISSUE THREAD

**Issue by bparees** [openshift/origin#17611]
@jwforres and @spadgett indicated that the ansible installer may not be setting the NODE_SELECTOR value, meaning the TSB pod will run everywhere:
https://github.com/openshift/origin/blob/master/install/templateservicebroker/apiserver-template.yaml#L18

the ansible installer should be setting the node selector to the infra nodes by default (and i guess allowing the admin to choose a different nodeselector as part of the install process).

/cc @jim-minter

**Comment by spadgett** [comment]
I didn't see it set here:

https://github.com/openshift/openshift-ansible/blob/master/roles/template_service_broker/tasks/install.yml#L43-L48

In fact, I don't see the parameter defined in the template in openshift-ansible, so they might be out of sync?

https://github.com/openshift/openshift-ansible/blob/master/files/origin-components/apiserver-template.yaml

**Comment by jim-minter** [comment]
@sdodson @ewolinetz I'm concerned that this is an issue that will need to be addressed in 3.7 - have a report on IRC:
```
<pschiffe> jminter: hi, I've deployed 3.7 with 3 masters, 3 infra and 3 app nodes, and openshift is trying to run apiserver pod on all 9 nodes (probably because of daemonset), but there is also Node-Selectors: role=app set for apiserver pod, so it fails on all nodes except the app nodes.
<pschiffe> jminter: now because of this, openshift is deleting and creating apiserver pods over and over again
```

**Comment by jtower525** [comment]
jim i have the same setup as pschiffe and opened a redhat case 01989456 earlier today
